lib/pages/guide/guide_page.dart

```diff
@@ -69,7 +76,10 @@ class _GuidePageState extends State<GuidePage> {
                         child: Padding(
                           padding: const EdgeInsets.only(left: 16),
                           child: Text(
-                            'Dieser Bereich wird in zukünftigen Versionen stetig ergänzt und um nützliche Hilfen wie bspw. einen interaktiven Raumfinder erweitert werden.',
+                            '''
```

Comment:
Sorry i overlooked it as it was not mentioned here. I messed up the String formatting because the Doku and explainations regarding multi line strings were a bit confusing. It now is a multi line String with the correct formatting as far as i can tell.